po/zh-CN.po

```diff
@@ -3610,13 +3641,18 @@ msgid ""
 "to the explicit declaration of a type.\n"
 "The compiler does the job for us and helps us write more concise code."
 msgstr ""
+"需要重点强调的是这样声明的变量并非像那种动态类型语言中可以持有任何数据的“任何类型”。"
+"这种声明所生成的机器码与明确类型声明完全相同。"
+"编译器进行类型推导只是能够让我们编写更简略的代码。"
```

Comment:
是否需要把原文的换行也加进来？

Reply:
我本地渲染了一下，看了一下原文的换行实际上并没有被渲染出来。
可以加上原文中的换行。

I rendered the doc locally. It seems that the original newline was not shown in the rendered page.
Sure we can add the newline here.

Reply:
> I rendered the doc locally. It seems that the original newline was not shown in the rendered page.

Perhaps I should add something about this to the [style guide](https://github.com/google/comprehensive-rust/blob/main/STYLE.md#translations)? The explanation is that you're editing Markdown, which means that a single newline shows up as a space in the rendered book. Two newlines would split the text into two paragraphs, which is very much visible :smile:

Reply:
> > I rendered the doc locally. It seems that the original newline was not shown in the rendered page.
> 
> Perhaps I should add something about this to the [style guide](https://github.com/google/comprehensive-rust/blob/main/STYLE.md#translations)? The explanation is that you're editing Markdown, which means that a single newline shows up as a space in the rendered book. Two newlines would split the text into two paragraphs, which is very much visible 😄

I think add a single newline here might not be a good idea. A space showed among the sentences.
It is OKAY in English because English uses space to separate words. But in Chinese, spaces are barely used.

I guess the newline here was trying to avoid a line being very long.

我觉得这里加一个换行符可能不好。这个换行符会导致这里出现一个空格。对于英语来说还好，因为英语本来就在单词之间使用空格。但是在中文很少有使用空格的情况。

我猜想这里使用换行符是为了避免原文中出现超长的一行。

Reply:
> 是否需要把原文的换行也加进来？

如上，我还是建议不要使用换行符。

As sentenced above, newlines here are not preferred.